### ISSUE TITLE
Your request: \'/_cluster/state/metadata/....\' is not allowed. AWS ES 2.3, curator 4.1.0

### ISSUE THREAD

**Issue by donbeave** [elastic/curator#770]
Can't use latest curator, which I have installed from pip on AWS Linux. My error log:
```
2016-09-26 18:39:40,349 DEBUG          curator.indexlist          __get_indices:65   Getting all indices
2016-09-26 18:39:40,357 DEBUG     requests.packages.urllib3.connectionpool          _make_request:387  "GET /_all/_settings?expand_wildcards=open%2Cclosed HTTP/1.1" 200 25798
2016-09-26 18:39:40,364 DEBUG              curator.utils            get_indices:384  All indices: [u'apache-app-2016.09.21', ..other indicies here... , u'request_logs-2016.09.23']
2016-09-26 18:39:40,364 DEBUG          curator.indexlist       empty_list_check:182  Checking for empty list
2016-09-26 18:39:40,365 DEBUG          curator.indexlist     __build_index_info:80   Building preliminary index metadata for apache-app-2016.09.21
....
2016-09-26 18:39:40,371 DEBUG          curator.indexlist     __build_index_info:80   Building preliminary index metadata for request_logs-2016.09.23
2016-09-26 18:39:40,371 DEBUG          curator.indexlist          _get_metadata:145  Getting index metadata
2016-09-26 18:39:40,372 DEBUG          curator.indexlist       empty_list_check:182  Checking for empty list
2016-09-26 18:39:40,378 DEBUG     requests.packages.urllib3.connectionpool          _make_request:387  "GET /_cluster/state/metadata/apache-app-2016.09.21,request_logs-2016.09.23,... HTTP/1.1" 401 3094
2016-09-26 18:39:40,383 ERROR                curator.cli                    cli:231  Failed to complete action: close.  <class 'elasticsearch.exceptions.AuthenticationException'>: TransportError(401, u'{"Message":"Your request: \'/_cluster/state/metadata/apache-app-2016.09.21,....\' is not allowed."}')
```

Running command:
```
/usr/local/bin/curator --config=$HOME/cfg $HOME/action
```

My `cfg` file:
```yaml
---
# Remember, leave a key empty if there is no value.  None will be a string,
# not a Python "NoneType"
client:
  hosts:
    - "blablabla.us-east-1.es.amazonaws.com"
  port: 443
  url_prefix:
  use_ssl: True
  certificate:
  client_cert:
  client_key:
  aws_key: "SOME_KEY_HERE"
  aws_secret_key: "SECRET_HERE"
  aws_region: "us-east-1"
  ssl_no_validate: False
  http_auth:
  timeout: 30
  master_only: False

logging:
  loglevel: DEBUG
  logfile:
  logformat: default
  blacklist: ['elasticsearch', 'urllib3']
```

My `action` file:
```yaml
---
# Remember, leave a key empty if there is no value.  None will be a string,
# not a Python "NoneType"
#
# Also remember that all examples have 'disable_action' set to True.  If you
# want to use this action as a template, be sure to set this to False after
# copying it.
actions:
  1:
    action: close
    description: >-
      Close indices older than 3 days (based on index name), for app-analytics-
      prefixed indices.
    options:
      delete_aliases: False
      timeout_override:
      continue_if_exception: False
      disable_action: False
    filters:
    - filtertype: pattern
      kind: prefix
      value: app-analytics-
      exclude:
    - filtertype: age
      source: name
      direction: older
      timestring: '%Y.%m.%d'
      unit: days
      unit_count: 3
      exclude:
```

**Comment by untergeek** [comment]
I am so sorry.  Your issue came just as I was pushing a the documentation change for #769 

The bottom line is that Curator cannot work with AWS ES because AWS ES does not support the `/_cluster/state/metadata` endpoint (see #685).  There is an AWS forum link in #685, and even a comment from one of the core AWS ES developers.

How did this happen?  Well, the `get_client` method is largely carried over from Curator 3.x, and some PRs were submitted to add AWS IAM functionality there.  Curator 4 was developed to support _only_ Elasticsearch 2.0 and higher.  At the time, AWS ES was only on version 1.5, so it was not compatible at all.  After Curator 4 was released, then Amazon released their 2.x version, and it did not support the `/_cluster/state/metadata` endpoint, which Curator 4 depends on.

I am improving the documentation in #771 to reflect these things.  I apologize for the inconvenience.

This ticket is being closed as a duplicate of #685

**Comment by donbeave** [comment]
Oh, sorry for the duplicate.
